glass-action/setup.py

```diff
@@ -13,7 +13,6 @@
       packages=setuptools.find_packages(),
       install_requires=[
           "click",
-          "PyGObject"
```

Comment:
Why did you remove this? Ideally all pip requirements should be installed by the setup.py:s, not Dockerfile, or stuff won't work _outside_ of docker.

Reply:
Done.